package.json

```diff
@@ -47,6 +48,8 @@
   },
   "scripts": {
     "build": "rm -rf dist && tsc && yarn build-storybook",
+    "deploy": "yarn docs && gh-pages -d build",
```

Comment:
Since `storybook-to-ghpages` is going to publish to gh pages, this command will "double publish" to gh pages. For now I think we should remove this.

Reply:
So, instead of

`"deploy": "yarn docs && gh-pages -d build",`

it should just say

`"deploy": "yarn docs",`?

Reply:
yea sounds good

Reply:
yea sounds good

Reply:
Okay, I changed it from 

`"deploy": "yarn docs && gh-pages -d build",`

to

`"deploy": "yarn docs",`